src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonClassInfo.cs

```diff
@@ -4,6 +4,7 @@
 
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Is this necessary (I didn't see any use of nullability attributes)?

Reply:
No - will remove. I used a nullability attribute in a helper in an earlier iteration of this change.